frontend/src/metabase/app.js

```diff
@@ -74,6 +74,9 @@ function _init(reducers, getRoutes, callback) {
     ] = MetabaseSettings.isTrackingEnabled() ? null : true;
   });
 
+  window.Metabase = window.Metabase || {};
+  window.Metabase.store = store;
```

Comment:
What's the idea behind having this on the window object? I'm a little hesitant to have the store exposed like this outside of a development env unless there's a good reason. I was able to call `getState` while logged out and since there was info in there from my previous session like the collection list, including sensitive info like my slack token. That's unlikely to happen unless a user was previously logged in on the same system, but I'm a little hesitant to open this up.

Reply:
This was for debugging/dev purposes. There's not really any additional security risk that wasn't already there. If I can run JavaScript in the same window I can almost certainly get a reference to the store.

Whether we should be resetting the store (or just refreshing the page) upon logout is a good question, though.